client/app/dedicatedCloud/dashboard/tiles/options/options.service.js

```diff
@@ -134,14 +134,23 @@ export const OptionsService = class OptionsService {
   getInitialData(serviceName, ovhSubsidiary, currentServicePackName) {
     return this
       .$q
-      .all({
-        pendingOrder: this
-          .getPendingOrder(serviceName),
-        currentOrFutureServicePack: this
-          .getCurrentOrFutureServicePack(serviceName),
-        servicePacks: this
-          .getServicePacks(serviceName, ovhSubsidiary),
-      })
+      .when(this
```

Comment:
You do need to use `$q.when` instead of directly return `getPendingOrder().then` ?

Reply:
Yes I do, it's a ES6 Promise ;)